mantidproject/mantid
# Add system tests for ZOOM TOML version conversion

**Description of work.**

Convert some existing legacy user files sent by the instrument
scientists into their respective ~SANS2D~ ZOOM versions
Edit: Waiting on SANS2D data first, so just zoom
**To test:**
- Check new system tests actually ran
- Check new docs for silly typos


There is no associated issue. Part of converting some representative files for the group

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
